Makefile

```diff
@@ -308,15 +323,17 @@ catalog-push: ## Push a catalog image.
 verify-manifests: manifests $(YQ) ## Verify manifests update.
 	git diff -I'^    containerImage:' -I'^        image:' --exit-code ./config
 	[ -z "$$(git ls-files --other --exclude-standard --directory --no-empty-directory ./config)" ]
-	yq ea -e 'select([.][].kind == "Deployment").spec.template.spec.containers[0].image | . == "$(OPERATOR_IMAGE)"' config/deploy/manifests.yaml
-	yq e -e '.metadata.annotations.containerImage == "$(OPERATOR_IMAGE)"' config/manifests/bases/authorino-operator.clusterserviceversion.yaml
+	$(YQ) ea -e 'select([.][].kind == "Deployment") | select([.][].metadata.name == "authorino-operator").spec.template.spec.containers[0].image | . == "$(OPERATOR_IMAGE)"' config/deploy/manifests.yaml
+	$(YQ) ea -e 'select([.][].kind == "Deployment") | select([.][].metadata.name == "authorino-webhooks").spec.template.spec.containers[0].image | . == "$(EXPECTED_DEFAULT_AUTHORINO_IMAGE)"' config/deploy/manifests.yaml
+	$(YQ) e -e '.metadata.annotations.containerImage == "$(OPERATOR_IMAGE)"' config/manifests/bases/authorino-operator.clusterserviceversion.yaml
 
 .PHONY: verify-bundle
 verify-bundle: bundle $(YQ) ## Verify bundle update.
-	git diff -I'^    containerImage:' -I'^                image:' --exit-code ./bundle
+	git diff -I'^    containerImage:' -I'^                    image:' --exit-code ./bundle
```

Comment:
```suggestion
	git diff -I' containerImage:' -I' image:' --exit-code ./bundle
```

Reply:
We can do it like this, but we'll start ignoring additional lines - for example the images in the scorecard folder would now match as well.

Reply:
No need. I see you've already fixed it.